Vizzuality/heco-invest
# Let 345 fe image uploader

This PR creates the file-uploader component and implements it on the project creation form using the direct-upload

* Adds component Uploader
* Modify the component ProjectImageGallery to allow deleting images
* Adds these components to project creation form
* Modify the imageUploader and Uploader import to next dynamic import
* Modify direct-upload service to directly import the activestorage library
* The loading bar is not implemented because it would take a lot of work. Please evaluate the need to include it in another task 

## Testing instructions
*  The style must conform to the [design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=1702%3A24167)
* User can upload up to six images, but the field is not mandatory
* The max size of each image should be 5MB
* The errors should be displayed below the input, and cleared with a new upload
* The form should be sent with the images

## Tracking

[LET-354](https://vizzuality.atlassian.net/browse/LET-345)
